Doc/c-api/contextvars.rst

```diff
@@ -139,12 +139,6 @@ Context object management functions:
    exception will be printed as an unraisable exception using
    :c:func:`PyErr_FormatUnraisable`. Otherwise it should return ``0``.
 
-   There may already be a pending exception set on entry to the callback. In
-   this case, the callback should return ``0`` with the same exception still
-   set. This means the callback may not call any other API that can set an
-   exception unless it saves and clears the exception state first, and restores
-   it before returning.
-
```

Comment:
Shouldn't we at least explain what happens if there is a pending exception set already? What does it do if the callback returns -1? What if it returns 0? I have no idea based on the rest of the description here. At the very least the previous paragraph should be rephrased to say something like "If the callback sets a new exception". (There's nothing saying what happens to the unraisable exception.)